solara/website/pages/__init__.py

```diff
@@ -95,7 +96,7 @@ def Layout(children=[]):
                 sub_title="Solara helps you build powerful & scalable Jupyter and web apps <b>faster</b> and <b>easier</b>.",
                 button_text="Quickstart",
             )
-
+"""
```

Comment:
why was this needed? Did you have trouble running this part? Is there something can do about it?
In any case, for this PR I think you want to revert this.

Reply:
@maartenbreddels yes, I had issues running the documentation* and ended up removing this files to run the documentation. This wasn't suppose to be deleted, it should be revered now.

*[I followed the setup docs](https://solara.dev/docs/development) but had problems running the tests and docs. I'll try to setup again later and open some issues if needed.